packages/nodejs/test/testHelpers.ts

```diff
@@ -97,3 +101,64 @@ export function getIdentityInput(): IdentityInput {
     };
     return identityInput;
 }
+
+type OverrideCheck<A> = {
+    when: (a: unknown) => boolean;
+    check: (l: A, r: A) => boolean;
+};
+
+const checkBuffers: OverrideCheck<ArrayBuffer> = {
+    when(u) {
+        return u instanceof ArrayBuffer;
+    },
+    check(l, r) {
+        if (l.byteLength !== r.byteLength) {
+            return false;
+        }
+        const lu8 = new Uint8Array(l);
+        const ru8 = new Uint8Array(r);
+        for (let i = 0; i < l.byteLength; i++) {
+            if (lu8.at(i) !== ru8.at(i)) {
+                return false;
+            }
+        }
+        return true;
+    },
+};
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const equalOverrides: OverrideCheck<any>[] = [
+    {
+        when: AccountAddress.isAccountAddress,
+        check: AccountAddress.equals,
+    },
+    { when: ContractAddress.isContractAddress, check: ContractAddress.equals },
+    checkBuffers,
+];
+
+export function expectToEqual<A = unknown>(value: A, expected: A) {
```

Comment:
All of these are only added to avoid migration, right? Maybe we should add some documentation stating this, or even move the functionality to a completely separate file, to avoid confusion?

Reply:
Thank you for pointing this out, since I ended up migrating all the test data, this is not really needed, so I have removed this again and while doing so found a bug in one of our types.